src/lib/stitching/lib/printType.ts

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
consolidate this, and handle `isInputObjectType`. basically adds `input: FilterArtworksInput` to all the places that stitch a filter connection in.